supervisor/ipsetctrl/acls.go

```diff
@@ -275,34 +275,39 @@ func (i *Instance) deleteIpsetOption(ip string) error {
 func (i *Instance) setupTrapRules(set string) error {
 
 	rules := [][]string{
-		// Application Syn and Syn/Ack in RAW
+		// Application Syn
 		{
 			i.appPacketIPTableContext, i.appPacketIPTableSection,
 			"-m", "set", "--match-set", set, "dst",
 			"-m", "set", "--match-set", containerSet, "src",
-			"-p", "tcp", "--tcp-flags", "FIN,SYN,RST,PSH,URG", "SYN",
-			"-j", "NFQUEUE", "--queue-balance", i.applicationQueues,
+			"-p", "tcp", "--tcp-flags", "SYN,ACK", "SYN",
```

Comment:
We should accept bad packets with syn such as syn with fin/rst/etc and drop them and report them.